### PR TITLE
RefreshSchedule job queue

### PR DESCRIPTION
**Pull request by sufeiiz**
Removed LoadConferenceDataTask, SeedScheduleDataTask. Added job queue for RefreshScheduleData.

### CODE REVIEW

File: app/src/main/java/co/touchlab/droidconandroid/shared/tasks/persisted/RefreshScheduleJob.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        CrashReport.logException(throwable);
+    }
+
+    @Override
+    protected RetryConstraint shouldReRunOnThrowable(@NonNull Throwable throwable, int runCount, int maxRunCount)
+    {
+        if(retryCount < 8)
+        {
+            retryCount++;
+            return RetryConstraint.createExponentialBackoff(runCount, DELAY_IN_MS);
```
Comment (samhill303) [inline app/src/main/java/co/touchlab/droidconandroid/shared/tasks/persisted/RefreshScheduleJob.java]: If our base delay is 5 minutes and we exponentially backoff 8 times, will the final delay be huge?

Comment (sufeiiz) [reply app/src/main/java/co/touchlab/droidconandroid/shared/tasks/persisted/RefreshScheduleJob.java]: You're right, I had it at 1min and I just changed it this morning - I forgot it's exponential.

Comment (sufeiiz) [reply app/src/main/java/co/touchlab/droidconandroid/shared/tasks/persisted/RefreshScheduleJob.java]: @samhill303  Changed back to 1min, max 8. Did some calculations and the max delay will be ~1hr. Is that too long? If we do max 7 max delay is ~30min.

Comment (samhill303) [reply app/src/main/java/co/touchlab/droidconandroid/shared/tasks/persisted/RefreshScheduleJob.java]: I think max 30 min would be more than enough.

**Review by samhill303 — APPROVED** [review]

**Review by candidcold — APPROVED** [review]
https://media.giphy.com/media/oBPOP48aQpIxq/giphy.gif

**Pull request merged**